docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
```

Comment:
A ToC would be nice.

Reply:
Also, the file is under `docs/` and starts with `8-`. What is the naming convention here?

Reply:
Added TOC

Reply:
They will be named in order following the blog series, but the troubleshooting guide seemed to be the most important one to write first. I know it looks a bit odd to have a folder with a single .md file named '8'. I'm open to removing the numbering and just naming them without numbers. What I don't want to do is change numbers later.

Reply:
I think we don't need numbers, wdyt @lamw ?
What would the doc name look like for the other docs to follow?